projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -516,7 +516,11 @@ export const isFeatureActive = ( state, featureSlug ) => {
 		case 'woocommerce':
 			return !! isPluginActive( state, 'woocommerce/woocommerce.php' );
 		case 'protect':
-			return !! isPluginActive( state, 'jetpack-protect/jetpack-protect.php' );
+			return (
+				!! isPluginActive( state, 'jetpack-protect/jetpack-protect.php' ) ||
+				!! isPluginActive( state, 'protect/jetpack-protect.php' ) ||
+				!! isPluginActive( state, 'jetpack-protect-dev/jetpack-protect.php' )
+			);
```

Comment:
Added alternate plugin names here so the feature check works in development. 
- In typical production sites, the plugin will be `jetpack-protect/jetpack-protect.php`.
- In the monorepo docker setup, the plugin will be `protect/jetpack-protect.php` due to the "protect" plugin directory symlink.
- Protect may be active via Jetpack Beta, which is `jetpack-protect-dev/jetpack-protect.php`